tests/migration_test.go

```diff
@@ -1055,7 +1055,10 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.UpdateClusterConfigValueAndWait("migrations", originalMigrationConfig)
 			})
 			It("[test_id:2227]should abort a vmi migration without progress", func() {
-				tests.IsRunningOnKindInfraIPv6()
+				// stress isnt enough in kind provider to cause the migration to fail,
```

Comment:
We don't know what is the root cause for the problem, kind/ipv6 or other. Please create a kubevirt issue (telling the test is not working and all the findings you have about it) and link the skip to the issue (you can ask @rmohr what's better - bugzilla or issue, I'm not sure).